world/console.cpp

```diff
@@ -406,6 +406,39 @@ void ConsoleGMSay(
 	zoneserver_list.SendChannelMessage(tmpname, 0, 11, 0, JoinString(args, " ").c_str());
 }
 
+/**
+ * @param connection
+ * @param command
+ * @param args
+ */
+void ConsoleGuildSay(
+	EQ::Net::ConsoleServerConnection *connection,
+	const std::string &command,
+	const std::vector<std::string> &args
+)
+{
+	if (args.size() < 1) {
+		return;
+	}
+
+	auto from = args[0];
+	auto guild_id = StringIsNumber(args[1]) ? std::stoul(args[1]) : 0;
+	if (!guild_id) {
+		return;
+	}
+
+	auto join_args = args;
+	join_args.erase(join_args.begin(), join_args.begin() + 2);
+
+	auto message = fmt::format(
+		"{} tells the guild, '{}'",
+		from,
+		JoinString(join_args, " ")
```

Comment:
Just a note for future use, fmt has a join method that you can also use without creating the `join_args` temporary
`fmt::join(args.begin()+2, args.end(), " ")`